pulpcore/cli/core/export.py

```diff
@@ -35,7 +41,7 @@ def read(pulp_ctx: PulpContext, export_ctx: PulpExportContext, href: str) -> Non
     pulp_ctx.output_result(entity)
 
 
-@pulp.command()
+@pulp.command(deprecated=True)
```

Comment:
Why deprecate these commands instead of just removing them?

Reply:
I wanted to get input from @ggainey at least.
Also the answer could be: "We want it to be called 'delete' everywhere."

Reply:
I am a huge fan of consistency. Also, I am/was not tied to the particular structure of the export-commands - I am perfectly fine with adjusting! And finally, I am **also** a big fan of using OO to alleviate boilerplate in every class. Def digging this whole PR.

Reply:
Yeah, that is mainly why i asked, because the "consistency" with the rest of the pulp ecosystem might suggest we call it `delete` instead of `destroy` for all resources.
I definitely want to have the same name in all parts of the CLI as a primary requirement.

Reply:
Right now it's largely just the three of us working on/with the CLI, I am ok with skipping deprecation cycles for now.